src/Stratis.Bitcoin.Features.PoA/PoAConsensusOptions.cs

```diff
@@ -19,6 +20,13 @@ public class PoAConsensusOptions : ConsensusOptions
         /// <summary>Adds capability of voting for adding\kicking federation members and other things.</summary>
         public bool VotingEnabled { get; protected set; }
 
+        /// <summary>Makes federation members kick idle members.</summary>
+        /// <remarks>Requires voting to be enabled to be set <c>true</c>.</remarks>
+        public bool AutoKickIdleMembers { get; protected set; }
+
+        /// <summary>Time that federation member has to be idle to be kicked by others in case <see cref="AutoKickIdleMembers"/> is enabled.</summary>
+        public uint FederationMemberMaxIdleTimeSeconds { get; protected set; }
```

Comment:
Should this be a number of "turns"?

Reply:
Not really. It's easier to keep it in seconds so when new network is created with different target spacing creator wont have to change this value.